config/server.js

```diff
@@ -0,0 +1,11 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
```

Comment:
This file already exists in the repo; it's not in the config dir, it's in the root dir - this can be deleted